starter-code/src/QueueDataStructure.js

```diff
@@ -1,2 +1,36 @@
-function QueueDataStructure () {
+function QueueDataStructure(){
+this.queueControl = [];
+this.MAX_SIZE = 4;
 }
+QueueDataStructure.prototype.isEmpty = function(){
+  if(this.queueControl.length == 0){
+    return true;
+  }else{
+    return false;
+  }
+};
+
```

Comment:
Nadia, está bien la comparación, pero acostúmbrate a utilizar comparaciones estrictas, es decir '===', se usan más cuando empieces a trabajar en una empresa.

Reply:
Por otro lado, la condición ya de por si es suficiente, ya que te devuelve true o false. No te haría falta un if o un else. Por ejemplo:

QueueDataStructure.prototype.isEmpty = function () {
  return this.stackControl.length === 0;
};